bundles/org.openhab.binding.bmwconnecteddrive/src/main/java/org/openhab/binding/bmwconnecteddrive/internal/dto/Destination.java

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.bmwconnecteddrive.internal.dto;
+
+import static org.openhab.binding.bmwconnecteddrive.internal.utils.Constants.*;
+
+import org.openhab.binding.bmwconnecteddrive.internal.utils.Constants;
+import org.openhab.binding.bmwconnecteddrive.internal.utils.Converter;
+
+/**
+ * The {@link Destination} Data Transfer Object
+ *
+ * @author Bernd Weymann - Initial contribution
+ */
+public class Destination {
+    public float lat;
+    public float lon;
+    public String country;
+    public String city;
+    public String street;
+    public String streetNumber;
+    public String type;
+    public String createdAt;
+
+    private String address = null;
+    private String coordinates = null;
```

Comment:
I feel it should be up to the callers of `getAddress()` and `getCoordinates()` to cache these values, not this class. Is there any performance reason to cache these values in this class instead?

Reply:
Not for performance reasons. The Handler is already quite packed and I want to provide easy calls which can be directly used to update the corresponding channles. An extra Wrapper for this small functionality isn't really needed from my point of view so I placed it here.